zkevm-contract-bindings/src/bin/deploy.rs

```diff
@@ -10,15 +10,17 @@ use async_compatibility_layer::logging::{setup_backtrace, setup_logging};
 use clap::Parser;
 use contract_bindings::HotShot;
 use ethers::{
+    prelude::{NonceManagerMiddleware, SignerMiddleware},
     providers::{Http, Middleware, Provider},
+    signers::{coins_bip39::English, MnemonicBuilder, Signer},
     types::Address,
     utils::{get_contract_address, parse_ether},
 };
 use hex::{FromHex, FromHexError};
-use sequencer_utils::{connect_rpc, Middleware as EthMiddleware};
```

Comment:
What was wrong with the `connect_rpc` from `sequencer_utils`?

Reply:
I just couldn't figure out how to change the polling interval after that function returns the client.